frontend/src/v5/ui/controls/assignees/assignees.component.tsx

```diff
@@ -40,11 +40,11 @@ export const Assignees = memo(({
 	disabled,
 	...props
 }: AssigneesProps) => {
-	const [values, setValues] = useState(initialValues);
-	const [open, setOpen] = useState(false);
-	const listedAssignees = initialValues.slice(0, max - 1);
-	const overflowAssignees = initialValues.slice(max - 1);
 	const allUsersAndJobs = UsersHooksSelectors.selectAssigneesListItems();
+	const [values, setValues] = useState(intersection(initialValues, allUsersAndJobs.map((i) => i.value)));
```

Comment:
can add a comment stating that initialValues could be invalid because of changes in the members of the teamspace?